server/src/main/java/org/apache/iotdb/db/engine/cache/LRULinkedHashMap.java

```diff
@@ -21,71 +21,56 @@
 
 import org.apache.iotdb.tsfile.common.cache.Accountable;
 
-import java.util.Iterator;
 import java.util.LinkedHashMap;
-import java.util.Map.Entry;
-import java.util.Set;
+import java.util.Map;
 
 /** This class is an LRU cache. <b>Note: It's not thread safe.</b> */
-public abstract class LRULinkedHashMap<K extends Accountable, V> {
+public abstract class LRULinkedHashMap<K extends Accountable, V> extends LinkedHashMap<K, V> {
```

Comment:
I think we should keep LinkedHashMap as a field instead of extending LinkedHashMap, because we don't need to expose all the methods of LinkedHashMap to users.

Reply:
Agree & Retract @JackieTien97